google/ggrc-core
# GGRC-4123 Delete records related to Clause

# Issue description
Delete all existing clause objects from database

# Steps to test the changes
Run migration and re-check database. Clause related objects and table "clauses" shouldn't be exist

# Solution description
Add migration for remove Clause related objects from database and table "clauses"

# Sanity checklist



# PR Review checklist
